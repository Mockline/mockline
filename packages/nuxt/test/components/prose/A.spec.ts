import { describe, it, expect } from 'vitest'
// @ts-ignore
import { MA } from '#components'
import type { TypeOf } from 'zod'
import ComponentRender from "../../component-render";

describe('ProseA', () => {
  it.each([
    ['basic case', {}]
  ])('renders %s correctly', async (nameOrHtml: string, options: TypeOf<typeof MA.props>) => {
    const html = await ComponentRender(nameOrHtml, options, MA)
    expect(html).toMatchSnapshot()
  })
})
