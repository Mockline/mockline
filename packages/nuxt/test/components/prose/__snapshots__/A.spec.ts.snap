// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProseA > renders basic case correctly 1`] = `"<a class="text-accent border-b border-transparent hover:border-accent font-medium focus-visible:outline-accent [&amp;>code]:border-dashed hover:[&amp;>code]:border-accent hover:[&amp;>code]:text-accent"></a>"`;
